lib/r10k/cli/deploy.rb

```diff
@@ -88,6 +88,7 @@ def self.command
 
           flag :p, :puppetfile, 'Display Puppetfile modules'
           flag nil, :detail, 'Display detailed information'
+          optional nil, :format, 'Display output in a specific format. Valid values: json, yaml', :argument => :required
```

Comment:
This can be `required` instead of `optional`, which drops the need to have `:argument => :required`